frontend/src/components/editor/output/MarimoErrorOutput.tsx

```diff
@@ -270,9 +278,24 @@ export const MarimoErrorOutput = ({
               </ul>
             </Fragment>
           ))}
+
+          <div>
+            Throwaway code?{" "}
+            <Button
+              size="xs"
+              variant="outline"
+              className="my-2 font-normal"
+              onClick={openScratchpad}
+            >
+              <NotebookPenIcon className="h-3 w-3 mr-2" />
+              Use the scratchpad
+            </Button>
```

Comment:
too many buttons?

Reply:
yea, think you're right. moved to a `<Tip>` and a link button variant.